01_install_requirements.sh

```diff
@@ -163,3 +163,10 @@ fi
 retry_with_timeout 5 60 "curl $OPENSHIFT_CLIENT_TOOLS_URL | sudo tar -U -C /usr/local/bin -xzf -"
 sudo chmod +x /usr/local/bin/oc
 oc version --client -o json
+
+if [[ -z "$OPENSHIFT_VERSION" ]]; then
+  source release_info.sh
+  save_release_info ${OPENSHIFT_RELEASE_IMAGE} ${OCP_DIR}
+  OPENSHIFT_VERSION=$(openshift_version)
+  export OPENSHIFT_VERSION
```

Comment:
This file is not sourced, so it's probably not useful.

Reply:
mmm right